exporters/dynamicconfig/confignotifier.go

```diff
@@ -35,17 +39,26 @@ func (config *MetricConfig) equals(otherConfig *MetricConfig) bool {
 	configString := fmt.Sprintf("%+v", *config)
 	otherConfigString := fmt.Sprintf("%+v", *otherConfig)
 
-	if configString == otherConfigString {
-		return true
-	}
-	return false
+	return configString == otherConfigString
 }
 
 // TODO: Read actual config from host
-// Right now we just return a fixed config
-func readConfig() *MetricConfig {
+// We don't do anything with the configHost string right now
+// TIMES_READ_TEST is for testing purposes, to mimic a dynamic service
+// The second time we call readConfig, the sampling period changes
+var TIMES_CONFIG_READ_TEST = 0
```

Comment:
Let's add a TODO to fix this later. A good approach is to passing a mock of config service.

Reply:
Is the TODO on [https://github.com/vmingchen/opentelemetry-go/blob/55cd0019794079ee2efbf498a1df25586daccecd/exporters/dynamicconfig/confignotifier.go#L45](line 45) enough, or should I add another TODO?